src/components/manage/Blocks/ToC/View.jsx

```diff
@@ -61,6 +64,25 @@ const View = (props) => {
           rootLevel = level;
         }
       }
+      if (isObject(entry) && isArray(entry.entries)) {
+        entry.entries.forEach((e, index) => {
+          const i = `${id}-${index}`;
+          const level = e[0];
+          const title = e[1];
+          const items = [];
+          if (!level || !levels.includes(level)) return;
+          tocEntriesLayout.push(i);
+          tocEntries[i] = {
+            level,
+            title: title || block.plaintext,
+            items,
+            id: i,
+          };
+          if (level < rootLevel) {
+            rootLevel = level;
+          }
+        });
+      }
```

Comment:
I don't think the changes in this file are needed. The new `tocEntry` function is always returning an array, not an object.